src/resources/postcss/tribe-events-full.pcss

```diff
@@ -2364,6 +2368,117 @@ input[name*='tribe-bar-']:-moz-placeholder {
 	}
 }
 
+/* Twenty Nineteen
+========================================================================== */
+.tribe-theme-twentynineteen {
+
+	h1.tribe-events-page-title::before {
```

Comment:
It looks like a common recurring thing with headings that the `::before` pseudo-element has a content that needs to be set to none. Rather than setting each one with a class, why don't we do something like:

```
h1,
h2,
h3,
h4,
h5,
h6 {

  &::before {
    content: none;
  }
}
```

or only the headings of concern (if we don't use h4 - h6 in our theme). If specificity is a concern then we can add more classes above this as necessary. This would also remove the repeated `::before` selectors below.

Reply:
great call!